packages/ui/react-ui-deck/src/components/PlankHeading/PlankHeading.tsx

```diff
@@ -31,6 +32,16 @@ const plankHeadingIconProps: IconProps = {
   weight: 'duotone',
 };
 
+type PlankRootProps = ThemedClassName<ComponentPropsWithRef<'div'>>;
```

Comment:
What is the relation between Deck, Stack, Section, and Plank?

Reply:
Stack has Sections, Deck has Planks, and they are otherwise unrelated as far as the UI is concerned.

I am amenable to renaming Plank to something else certainly.